extension.js

```diff
@@ -22,15 +22,23 @@ let grab_window = null;
 const APP_ID = 'com.github.amezin.ddterm';
 const APP_DBUS_PATH = '/com/github/amezin/ddterm';
 const WINDOW_PATH_PREFIX = `${APP_DBUS_PATH}/window/`;
-const SUBPROCESS_ARGV = [Me.dir.get_child('com.github.amezin.ddterm').get_path(), '--undecorated'];
+const SUBPROCESS_ARGV = [
+    Me.dir.get_child('com.github.amezin.ddterm').get_path(),
+    '--undecorated',
+];
```

Comment:
Please do not reformat the code unnecessarily. It only creates additional conflicts when merging/rebasing.